docs/troubleshooting.md

```diff
@@ -109,20 +118,26 @@ If everything seems healthy, and still no packages show up in the UI, it could b
 
 ## OperatorSource failing to download from datastore
 
-If an OperatorSource fails to reconcile for any reason (i.e operators in that particular OperatorSource fail to show up in the UI), it might be helpful first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator for detailed information about the OperatorSource. 
+If an OperatorSource fails to reconcile for any reason, first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator with `oc logs <name-of-marketplace-operator> -n openshift-marketplace` for detailed information about the OperatorSource. 
 
-Note: A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [endpoint](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the endpoint is specified as `https://quay.io/cnr`. 
 
 ## OperatorSource `MESSAGE` reads unknown error (status 404)
 
-Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong end point being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
+Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong endpoint being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
```

Comment:
What is the difference between section `OperatorSource failing to download from datastore` and `OperatorSource `MESSAGE` reads unknown error (status 404)`. I suggest collapsing them into a single section, `OperatorSource failing to connect to the datastore`, calling out `status 404` as one of the error cases.